src/main/java/network/brightspots/rcv/TallyTransfers.java

```diff
@@ -55,11 +75,11 @@
   void addTransfer(int round, String sourceCandidate, String targetCandidate, BigDecimal value) {
     // null source means we are transferring the initial count
     if (sourceCandidate == null) {
-      sourceCandidate = "uncounted";
+      sourceCandidate = UNCOUNTED;
```

Comment:
Minor alarm bells going off with setting directly equal to a declared string. It points to the object rather than the value. But I guess that's ok here?

Reply:
I tested to confirm that this isn't an issue. This line makes `sourceCandidate` a pointer to the value, not to `UNCOUNTED`. Even if `UNCOUNTED` were to change, `sourceCandidate` would remain the same.